verification/curator-service/ui/Dockerfile

```diff
@@ -20,5 +20,4 @@ EXPOSE 3002
 # Start service, will trigger a build.
 # Environment variables must be passed, notably:
 # - PORT
-# - REACT_APP_API_ENDPOINT
```

Comment:
This is now REACT_APP_PROXY_URL, right?

Reply:
ah yeah sorry I did that before adding a default value that doesn't work with docker, added back.